ui/app/templates/components/transit-key-action/verify.hbs

```diff
@@ -192,16 +192,25 @@
     </div>
   </div>
 </form>
-<Modal @title="Results" @onClose={{action (mut @isModalActive) false}} @isActive={{@isModalActive}}>
```

Comment:
Previously the modal didn't even have a close button! 🤯 
<img width="814" alt="Screenshot 2023-09-28 at 2 05 35 PM" src="https://github.com/hashicorp/vault/assets/68122737/39dbc38f-edfc-42dc-a2ee-470ec54a12f9">
<img width="884" alt="Screenshot 2023-09-28 at 2 05 49 PM" src="https://github.com/hashicorp/vault/assets/68122737/3d9de2f6-d4ca-4e08-a3a8-c8947dd617a8">